### ISSUE TITLE
No module named quark_hash

### ISSUE THREAD

**Issue by tincturaginseng** [Neisklar/p2pool-quarkcoin#1]
When I run p2pool with runnind quarkcoind I get this errors:

2013-09-13 13:31:14.047157 Initializing work...
2013-09-13 13:31:14.145307 RECV headers 0102000000b056ae1a289cd59ae04afcb053dd7d4ce8dac9729cb45e9d2cc8b2510000000005c99d8030246546b95136635aa5ad3f351ca71ce5ba6b52e92bf0279e941a59b8db3252328c001dd001b16600
2013-09-13 13:31:14.146922 > Error handling message: (see RECV line)
2013-09-13 13:31:14.147102 > Traceback (most recent call last):
2013-09-13 13:31:14.147301 >   File "/usr/lib/python2.7/dist-packages/twisted/internet/tcp.py", line 215, in doRead
2013-09-13 13:31:14.147471 >     return self._dataReceived(data)
2013-09-13 13:31:14.147641 >   File "/usr/lib/python2.7/dist-packages/twisted/internet/tcp.py", line 221, in _dataReceived
2013-09-13 13:31:14.147860 >     rval = self.protocol.dataReceived(data)
2013-09-13 13:31:14.148050 >   File "/home/qrkminer/p2pool-quarkcoin/p2pool/util/p2protocol.py", line 27, in dataReceived
2013-09-13 13:31:14.148199 >     self.dataReceived2(data)
2013-09-13 13:31:14.148366 >   File "/home/qrkminer/p2pool-quarkcoin/p2pool/util/datachunker.py", line 40, in _DataChunker
2013-09-13 13:31:14.148535 >     wants = receiver.send(buf.get(wants))
2013-09-13 13:31:14.148698 > --- <exception caught here> ---
2013-09-13 13:31:14.148892 >   File "/home/qrkminer/p2pool-quarkcoin/p2pool/util/p2protocol.py", line 57, in dataReceiver
2013-09-13 13:31:14.149132 >     self.packetReceived(command, type_.unpack(payload, self.ignore_trailing_payload))
2013-09-13 13:31:14.149366 >   File "/home/qrkminer/p2pool-quarkcoin/p2pool/util/p2protocol.py", line 71, in packetReceived
2013-09-13 13:31:14.149647 >     handler(**payload2)
2013-09-13 13:31:14.149854 >   File "/home/qrkminer/p2pool-quarkcoin/p2pool/bitcoin/p2p.py", line 130, in handle_headers
2013-09-13 13:31:14.150155 >     header_hash = self.net.BLOCKHASH_FUNC(bitcoin_data.block_header_type.pack(header))
2013-09-13 13:31:14.150389 >   File "/home/qrkminer/p2pool-quarkcoin/p2pool/bitcoin/networks.py", line 25, in <lambda>
2013-09-13 13:31:14.150629 >     BLOCKHASH_FUNC=lambda data: pack.IntType(256).unpack(**import**('quark_hash').getPoWHash(data)),
2013-09-13 13:31:14.150869 > exceptions.ImportError: No module named quark_hash

Where is module quark_hash?

**Comment by Neisklar** [comment]
It's in a seperate repo (since it is also used in the block exlorer)
https://github.com/Neisklar/quarkcoin-hash-python

**Comment by fasbit** [comment]
1. I got the same error as OP
2. I git cloned the above repo into the top level folder of the target p2pool folder, but it was still a "no go"
   What do I do with the "https://github.com/Neisklar/quarkcoin-hash-python" repo?   There are no installation instructions.

Solved my Own Question:
Edit:  In case any else gets this error:  clone the repo in to the top level directory, cd into the new directory, then $ sudo python setup.py install - works like a champ!

./fasbit
